rust/src/measurements/make_private_expr/expr_report_noisy_max/mod.rs

```diff
@@ -295,6 +296,15 @@ fn report_noisy_max_gumbel_udf(
                         PolarsError::InvalidOperation("input dtype does not match".into())
                     })?;
 
+                // when all scores are same, return a random index
+                // this is a workaround for slow performance of the gumbel mechanism
+                // when all scores are the same
```

Comment:
```suggestion
                // When all scores are same, return a random index.
                // This is a workaround for slow performance of the gumbel mechanism
                // when all scores are the same.
```
Just a little more readable.